packages/data/walkthroughs/9.9.9.json

```diff
@@ -1555,6 +1555,326 @@
+        },
+        {
+          "answerDisplayText": "Greater than 2",
+          "answerValue": "3"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P12"
+        }
+      ]
+    },
+    "P12": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Is this <defined-term>dwelling unit</defined-term> provided with at least two separate <defined-term>means of egress</defined-term>?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P12",
+          "answerValue": "true",
+          "nextNavigateTo": "R10"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P13"
+        }
+      ]
+    },
+    "P13": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Does this <defined-term>dwelling unit</defined-term>’s egress door open onto a <defined-term>public corridor</defined-term> or an exterior passageway?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P13",
+          "answerValue": "false",
+          "nextNavigateTo": "P17"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P14"
+        }
+      ]
+    },
+    "P14": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Where the egress door opens onto the corridor or exterior passageway, is it possible to go in opposite directions to 2 separate <defined-term>exits</defined-term>?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P14",
+          "answerValue": "true",
+          "nextNavigateTo": "P17"
+        },
+        {
+          "nextLogicType": "greaterThan",
+          "answerToCheck": "P11",
+          "answerValue": "2",
+          "nextNavigateTo": "R20"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P15"
+        }
+      ]
+    },
+    "P15": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Does the egress door of this <defined-term>dwelling unit</defined-term> open onto a corridor that meets the requirements of Sentence 9.9.7.3.(1)?",
```

Comment:
@laurenrevere We still don't have an exact solution for this case (link to out of scope Building Code area). My thoughts are:

Easy: Is a link that downloads the PDF.
Nice to have: Link contains a "named destination" like this link:
https://www2.gov.bc.ca/assets/gov/farming-natural-resources-and-industry/construction-industry/building-codes-and-standards/revisions-and-mo/bcbc_2024.pdf#nameddest=9.9.9.%20Egress%20from%20Dwelling%20Units

This would jump the user to that section of the document. But only in certain browsers and PDF readers that support this functionality.